MesaModel/run.py

```diff
@@ -27,6 +27,12 @@
     default=f"../classes_output/output{datetime.datetime.now().strftime('%Y-%m-%d_%H%M%S')}.csv",
     help="Output file path, relative to current working directory",
 )
+@click.option(
+    "--n-processors",
+    "-npr",  # 'np' is avoided given the meaning this already has in MPI contexts
```

Comment:
Could we just use `-n` or `-p` for brevity?

Reply:
-p would do the trick.